code/modules/fishing/sources/source_types.dm

```diff
@@ -158,6 +158,7 @@
 	radial_name = "Chasm"
 	overlay_state = "portal_chasm"
 	radial_state = "ground_hole"
+	fish_source_flags = FISH_SOURCE_FLAG_EXPLOSIVE_NONE
```

Comment:
I was talking about the actual chasms, not the portal.

Beside, you should remove the `spawn_reward_from_explosion` overrides from the fish source datums for chasms and the aquarium since they're redundant.